--- post.py	2023-11-09 19:56:28.379241491 +0530
+++ post.py	2023-11-09 20:31:28.547995042 +0530
@@ -463,7 +463,7 @@

 def mk_relative_osx(path, host_prefix, m, files, rpaths=("lib",)):
     base_prefix = m.config.build_folder
-    assert base_prefix == dirname(host_prefix)
+
     build_prefix = m.config.build_prefix
     prefix = build_prefix if exists(build_prefix) else host_prefix
     names = macho.otool(path, prefix)
